cli/data.py

```diff
@@ -30,6 +30,18 @@
 import pyseir.icu.utils
 from pyseir.icu import infer_icu
 
+CUMULATIVE_FIELDS_TO_FILTER = [
+    CommonFields.CASES,
+    CommonFields.DEATHS,
+    CommonFields.POSITIVE_TESTS,
+    CommonFields.NEGATIVE_TESTS,
+    CommonFields.TOTAL_TESTS,
+    CommonFields.POSITIVE_TESTS_VIRAL,
+    CommonFields.POSITIVE_CASES_VIRAL,
+    CommonFields.TOTAL_TESTS_VIRAL,
+    CommonFields.TOTAL_TESTS_PEOPLE_VIRAL,
+    CommonFields.TOTAL_TEST_ENCOUNTERS_VIRAL,
```

Comment:
I realize this isn't a cumulative field, but do we want to filter on TEST_POSITIVITY_7D and TEST_POSITIVITY_14D?

Reply:
Sure, but as you say, this filter is for cumulative metrics, not ratios. Are you thinking of something like 'guardrails' that trigger when it is outside a sensible range (0.1% - 60%) ?

Reply:
Yeah and actually we already have the filter that removes trailing zeros in covid-data-public so I don't think it's necessary here